### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by joogps**
This pull request adds translations for Portuguese (I speak Brazilian Portuguese).

I don't really use Broadcasts (now I decided to download it), it's just that I follow you and this repo appeared in my feed. Thought I could contribute somehow. :)

### CODE REVIEW

File: pt.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/* 
+  Localizable.strings
+  MobileRadio
+
+  Created by João Gabriel Pozzobon dos Santos on 05/08/2020.
```
Comment (joogps) [inline pt.lproj/Localizable.strings:5]: Did I do this right?

**Comment by steventroughtonsmith** [comment]
Hi João! Thanks for this! Since this is Brazilian Portuguese, if you could rename the pt.lproj folder to pt-BR.lproj that would be great. I'll get this double-checked in the meantime and if all's good will make sure it's included in the next update :D

**Comment by steventroughtonsmith** [comment]
Hey João; just checking — any chance you could provide a translation for UPSELL_COPY and UPSELL_BUY? Everything else looks great in the Simulator. Thanks!

Simulator Screen Shot - iPad Pro (11-inch) - 2020-05-09 at 01 51 43

**Comment by joogps** [comment]
I completely missed that! Will make a PR soon

**Comment by steventroughtonsmith** [comment]
Awesome, thanks! 🙏

**Comment by joogps** [comment]
Also, what rule should I use for capitalization? In Portuguese we don't really have a standard for that (I would suggest to have only the first letter capitalized)

**Comment by steventroughtonsmith** [comment]
If in doubt, do it the way you see Apple doing it in their apps. Apps like Notes and Pages have a welcome screen that might help

**Comment by joogps** [comment]
Pull request created! #6

**Pull request merged**